APIs_Drivers/LowPowerTicker_HelloWorld/main.cpp

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
could you rename the folder to the example?
something like `lowpowerTicker_ex1`?